docs/content/monitoring.md

```diff
@@ -74,6 +74,23 @@ The Prometheus endpoint exports Go runtime metrics as well as HTTP request laten
 | go_threads | gauge | Number of OS threads created. |
 | http_request_duration_seconds | histogram | A histogram of duration for requests. |
 
+
+### Status Metrics
```

Comment:
It would be helpful to provide a description  in the status page of how these metrics would be laid out.

Reply:
Hey @ashutosh-narkar I’ve just updated the management-status doc with the prometheus section ;) I'm planning to create a grafana dashboard and commit it as an example. What do you think about that?

Reply:
> I'm planning to create a grafana dashboard and commit it as an example.

I think this would be a great addition to the contrib repo, https://github.com/open-policy-agent/contrib 👍

Reply:
Would you mind adding a "Status" column similarly to the example you've shared above, https://github.com/kubernetes/kube-state-metrics/blob/master/docs/cronjob-metrics.md? I think introducing the new metrics as "Experimental" would be a good first step:

1. we'll have some wiggle room to change them if need be
2. anyone using the new metrics will feel encouraged to give their feedback

What do you think?

Reply:
Good point, I will do that.

Reply:
Thank you!